### ISSUE TITLE
basic tests for cloudabi-utils

### ISSUE THREAD

**Issue by mcandre** [NuxiNL/cloudabi-utils#8]
Could we add some basic command line tests for cloudabi-run and so on, to ensure that the utilities match the advertised capabilities as described in the man pages?

https://gitlab.kitware.com/cmake/community/wikis/doc/ctest/Testing-With-CTest

**Comment by EdSchouten** [comment]
Yes, we should add more testing coverage. The issue you ran into, that `!fd <number>` didn't work, could have had a corresponding unit test to ensure this never regresses going forward.

Now that I'm working towards having Bazel build rules for CloudABI's components, the barrier towards adding unit tests to the yaml2argdata repository should be far lower than before. I'll try to add some unit testing coverage the next time I need to touch this library.
